.github/workflows/tests.yaml

```diff
@@ -1,9 +1,16 @@
 name: Tests build
 
-on: [push, pull_request]
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
 
 jobs:
   lint:
+    if: ${{ github.repository }} == 'iteratie/dvc'
```

Comment:
I don't quite understand why this change was necessary, as the workflow for forks run on their own repository.
There is a spelling error too. `s/iteratie/iterative`.

Reply:
So, since we promote PR's from users repositories, it is just wasteful to run full builds. Typo is a good point, and since the recent builds are still building, it seems like its not respected. Will get rid of that.